build.gradle

```diff
@@ -29,6 +29,21 @@ allprojects {
 
 apply plugin: 'java'
 
+/* SET PROJECT INFO for all artifacts */
+assert hasProperty('version')   // See version in gradle.properties.
+System.out.println("version=${version}")
+group = 'org.apache.edgent'
+
+def getDate() {
```

Comment:
I don't understand why getData() was added / how it's used.  Do other things use the ext.buildDate property that it establishes, or do other things invoke getDate() if its defined?